content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -254,7 +254,7 @@ const user = await prisma.user.findFirst({
 
 Note that you **cannot** use `select` and `include` _on the same level_. This means that if you choose to `include` a user's post and `select` each post's title, you cannot `select` only the users' `email`:
 
-<CodeWithResult>
+<CodeWithResult outputResultText="">
```

Comment:
Does not seem to work? 
https://docs-git-jharrell-fix-codewithresult-1-prisma.vercel.app/concepts/components/prisma-client/relation-queries#:~:text=%7D%2C-,%7D),-Hide%20CLI%20results

Reply:
Ugh you're right. `""` is falsy so this won't work.

Reply:
Going to punt on this for now. We might want to update the component in the future to allow for this sort of thing.